submissions/kt-std/tiny-js-world/index.js

```diff
@@ -0,0 +1,57 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+   
+   function Inhabitant (species, name, gender, hands, legs, sayings, friends){
+   		this.species = species;
+   		this.name = name;
+   		this.gender = gender;
+   		this.hands = hands === 0 ? `${this.species}s don't have hands! They have paws`: hands;
+   		this.legs = legs;
+   		this.friends = Array.isArray(friends) && friends.length ? friends : 'Forever alone...';
+   		this.saySomething = sayings;
+   }
+
+   const dog = new Inhabitant('dog', 'Rex', 'male', 0, 4, 'Woof!', ['Mukhtar', 'Sharik', 'Thunder']);
+   const cat = new Inhabitant('cat', 'Murka', 'female', 0, 4, 'Meow!', ['Murchik', 'Anna']);
+   const woman = new Inhabitant('woman', 'Anna', 'female', 2, 2, 'Hi, folks!', ['John', 'Tom']);
+   const man = new Inhabitant('man', 'John', 'male', 2, 2, 'Ahoy!');
+   const catWoman = new Inhabitant('superhero', 'Selina', 'female?', 2, 2, cat.saySomething, ['Alice', 'Arizona']);
+
+   ((array) => {
+	   array.forEach( obj => {
+	   	let string = '';
+	   	for(prop in obj){
+	   		Array.isArray(obj[prop])? string += `${obj[prop].join(", ")}; ` : string += `${obj[prop]}; `;
```

Comment:
Order of keys in object not guaranteed by design. Please use an explicit list of properties.
Consider also employing `Array` methods `map` and `join`

Reply:
I updated the the code with explicitly set order of object properties and applied `map` method in conjunction with `join`.